docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,239 @@
+#####################################################
+Compute circuit output probabilities with ``Sampler``
+#####################################################
+
+This guide shows how to get the probability distribution of a quantum circuit with the :class:`~qiskit.primitives.Sampler` primitive.
+
+.. note::
+
+    While this guide only uses Qiskit Terra's implementation of the ``Sampler`` primitive, there are other
+    implementations of this primitive like Qiskit Terra's :class:`~qiskit.primitives.BackendSampler`, Qiskit Aer's :class:`~qiskit_aer.primitives.Sampler`
+    and Qiskit Runtime's :class:`~qiskit_ibm_runtime.Sampler`.
```

Comment:
```suggestion
While this guide uses Qiskit’s reference implementation, the ``Sampler`` primitive can be run with any provider using :class:`~qiskit.primitives.BackendSampler` .
```

Reply:
We are going to add link to the new providers page which will contain a section of providers that have native primitive support and the link will be http://qiskit.org/providers#native_primitives

Reply:
Put the PR on hold until the providers page is launched, which should be in the next few days.